docs/src/misc.rst

```diff
@@ -125,6 +125,36 @@ Data types
             } netmask;
         } uv_interface_address_t;
 
+.. c:type:: uv_network_interface_t
+
+    Data type for network interfaces.
+
+    ::
+
+        typedef struct uv_network_interface_s {
+            char* name;
+            int is_up_and_running;
+            int is_loopback;
+            int is_point_to_point;
+            int is_promiscuous;
+            int has_broadcast;
+            int has_multicast;
+            char phys_addr[8]; /* Enough to store a firewire address. */
+            unsigned int phys_addr_len;
+            union {
+                struct sockaddr_in address4;
+                struct sockaddr_in6 address6;
+            } address;
+            union {
+                struct sockaddr_in broadcast4;
+                struct sockaddr_in6 broadcast6;
+            } broadcast;
+            union {
+                struct sockaddr_in netmask4;
+                struct sockaddr_in6 netmask6;
```

Comment:
If this is a breaking change, I'd suggest to do away with these IPv6 "netmasks". IPv6 does not use netmasks but prefixes. I'd say to have a unified interface for both v4 and v6, it'd be better to provide a combined CIDR form.